devutils/validate_patches.py

```diff
@@ -661,10 +661,7 @@ def main():
     if not args.patches.is_dir():
         parser.error('--patches path is not a directory or not found: {}'.format(args.patches))
 
-    if args.verbose:
-        get_logger(initial_level=logging.DEBUG)
-    else:
-        get_logger(initial_level=logging.INFO)
+    set_logging_level(verbose=args.verbose, quiet=args.quiet)
```

Comment:
Instead of calling `set_logging_level` everywhere, I'm wondering if there's a more concise way to do this, like:

1. Pass `args` into a function that handles all common arguments, e.g. `process_common_params`. We can call `set_logging_level` in this new function.
2. Create a subclass of `Action` and set it during the `add_argument` invocation in `add_common_params`. That way, we don't even need to invoke `process_common_params` in all the callbacks. This will also discourage interdependency between flags, which I think is an anti-pattern.

Reply:
Ah, didn't know there is an `Action` class. Then it will be much clear to use this class instead of a global function.